src/ui/Logic/VideoPlayers/LibVlcDynamic.cs

```diff
@@ -159,7 +158,7 @@ public class LibVlcDynamic : VideoPlayer, IDisposable
         private libvlc_media_player_next_frame _libvlc_media_player_next_frame;
 
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-        private delegate int libvlc_media_player_add_slave(IntPtr media, int type, byte[] filePath, bool select);
+        private delegate int libvlc_media_player_add_slave(IntPtr mediaPlayer, int type, byte[] filePath, bool select);
```

Comment:
And by the way, this is "mediaPlayer", the one in "media" has different parameters.
There is no `Clear` method for input slave in "media player", but there is in "media".
That's why I couldn't get the "Close second subtitle" to work on this.